lib/api.rb

```diff
@@ -261,7 +261,7 @@ def self.relation_for_sis_mapping_and_columns(relation, columns, sis_mapping, si
   end
 
   def self.max_per_page
-    Setting.get('api_max_per_page', '50').to_i
+    Setting.get('api_max_per_page', '500').to_i
```

Comment:
Big jump, wonder what impact a request for 500 would have.  I prob would have gone with something more like 200 since that is the max of an item that I can envision being viewed on a page. But whatev's, I assume we're not really going to be calling the API with requests this large, except in rare cases of backend cases, so I think it's fine to have the flexibility.